src/js/helpers/Repo.js

```diff
@@ -11,6 +11,18 @@ import GitApi, {
   getSavedRemote,
 } from './GitApi';
 
+// consts
+export const GIT_ERROR_UNKNOWN_PROBLEM = 'An unknown problem occurred during import';
```

Comment:
Should the text be localized?

Reply:
UI text is localized in src/js/actions/ProjectUploadActions.js, see gitErrorToLocalizedPrompt().